activerecord/lib/active_record/timestamp.rb

```diff
@@ -94,11 +97,11 @@ def all_timestamp_attributes_in_model
     end
 
     def timestamp_attributes_for_update
```

Comment:
are these intended to be user-overridable?

if yes, should you use these methods rather than constants in https://github.com/rails/rails/pull/21979/files#diff-ec4e6359a60e5d11bbc4bd0912265c27R87 ? if not, why not just remove the methods and use the constants directly?